### ISSUE TITLE
Log files do not contain timestamps

### ISSUE THREAD

**Issue by meersjo** [duplicati/duplicati#2846]
- [X] I have searched open and closed issues for duplicates.

----------------------------------------

## Environment info
- **Duplicati version**: 2.0.2.9-1
- **Operating system**: Linux Mint 17.3 (≃ Ubuntu Trusty)
- **Backend**: SFTP

## Description
Log files do not contain timestamps. Just a regular date/time stamp would be good; including hostname and process info would be better - why not use the syslog facilities on the respective operating systems?

- **Actual result**:
No timestamp in logs:
```
Server has started and is listening on 127.0.0.1, port 8200
```
- **Expected result**:
```
Oct 25 11:40:09 lappie duplicati-server[2004]: Server has started and is listening on 127.0.0.1, port 8200
```

**Comment by kenkendk** [comment]
I agree, we should use the system log facilities. I would like to move to log4net, and I have used the syslog plugins for it in other projects.
